addons/mrp/tests/test_warehouse_multistep_manufacturing.py

```diff
@@ -609,3 +609,42 @@ def test_child_parent_relationship_on_backorder_creation(self):
         self.assertEqual(mo.mrp_production_child_count, 0, "Children MOs counted as existing where there should be none")
         self.assertEqual(mo.mrp_production_source_count, 0, "Source MOs counted as existing where there should be none")
         self.assertEqual(mo.mrp_production_backorder_count, 2)
+
+    def test_split_merge_3_steps(self):
+        """Check that default location comes same after splittin mo
+        when operation type is 3-step manufacturers are selected."""
```

Comment:
English fix:

"""Check that default values are correct after merging mos when 3-step manufacturing"""

You are also not splitting anything in this test so you should either add a split to the test (preferred since you are also fixing something with the split, but not testing that fix) or remove it from the name of the test. If you do add a split to your test, remember to update your test description to match.